cli/src/main/kotlin/commands/DownloaderCommand.kt

```diff
@@ -51,7 +51,7 @@ object DownloaderCommand : CommandWithHelp() {
     @Parameter(
         description = "An ORT result file with an analyzer result to use. Must not be used together with " +
                 "'--project-url'.",
-        names = ["--ort-file", "-a"],
+        names = ["--ort-file", "-i"],
```

Comment:
Please also align this in https://github.com/heremaps/oss-review-toolkit/blob/ea802a097615098bc50142d05f73da3eac5cf91b/Jenkinsfile#L80

Reply:
Done.